gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaAvroJobStatusMonitor.java

```diff
@@ -160,6 +160,7 @@ public GobblinTrackingEvent deserializeEvent(DecodeableKafkaRecord<byte[],byte[]
         break;
       case TimingEvent.FlowTimings.FLOW_CANCELLED:
       case TimingEvent.LauncherTimings.JOB_CANCEL:
+      case TimingEvent.JOB_SKIP_TIME:
         properties.put(JobStatusRetriever.EVENT_NAME_FIELD, ExecutionStatus.CANCELLED.name());
```

Comment:
Should we introduce a new ExecutionStatus for SKIPPED JOBS.
This way we can differentiate between these two events and debug accordingly if needed.
Also cancelled may be retried in some cases, but skipped may not be. @phet  I vaguely remember we discussed this some day?

Reply:
Yes, I do agree that it's better to introduce another status call SKIPPED, but that will require multiple change in job status monitor, status retriever, job execution pdl and data template as well as the logic in DAG manager to deal with this new status (we even treat cancel same as failure now in DM). Since it's a fairly large change and require more testing, I will create a separate ticket for that. For now, I would prefer to leverage the existing function to make sure the DM will not block long running jobs in gobblin cluster.

Reply:
To add on this, I try to make the change to include new status, and it requires me to add one symbol to ExecutionStatus.pdl, and that a backward incompatible change and cannot pass the compilation. I check the doc here https://linkedin.github.io/rest.li/modeling/compatibility_check#why-is-adding-to-an-enum-considered-backwards-incompatible and seems we need to do a migration to make this happen. Thus again, I'd prefer to do it in a separate ticket.

Reply:
I definitely agree with a distinct status... and also see the complication, so can understand the consideration to follow up.  to get more specific, how long do we expect between this change and that new status?  do you propse an immediate follow-on or something we might not prioritize for a month or more?